packages/ui/src/components/app-sidebar/sidebar-user.tsx

```diff
@@ -48,61 +47,32 @@ export function User({ user, openThemeDialog, openLanguageDialog, handleLogOut }
             align="end"
             sideOffset={4}
           >
-            <DropdownMenu.Label className="p-0 font-normal">
-              <div className="flex items-center gap-2 px-1 py-1.5">
-                <UserAvatar user={user} />
-              </div>
-            </DropdownMenu.Label>
+            <DropdownMenu.Header className="flex items-center gap-2 p-0 px-1 py-1.5 font-normal">
```

Comment:
why do we need custom css here (and many other places). Are the screen designs not following the standard component design? 
I believe that since the component was standardised after the initial screens, we should just use the component as-it-is, and avoid adding overrides.